### PR TITLE
miniaudicle: init at 1.3.5.2

### PR DESCRIPTION
**Pull request by fgaz**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by zowoq** [comment]
@ofborg eval

File: pkgs/applications/audio/miniaudicle/default.nix [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+, libjack2
+, audioBackend ? "pulse" # "pulse", "alsa", or "jack"
+}:
+
+stdenv.mkDerivation rec {
+  pname = "miniaudicle";
+  version = "1.3.5.2";
+
+  src = fetchurl {
+    url = "http://audicle.cs.princeton.edu/mini/release/files/miniAudicle-${version}.tgz";
```
Comment (SuperSandro2000) [inline pkgs/applications/audio/miniaudicle/default.nix:21]:
```suggestion
    url = "https://audicle.cs.princeton.edu/mini/release/files/miniAudicle-${version}.tgz";
```

File: pkgs/applications/audio/miniaudicle/default.nix [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    ++ lib.optional (audioBackend == "jack")  libjack2;
+
+  buildFlags = [ "linux-${audioBackend}" ];
+
+  makeFlags = [ "PREFIX=$(out)" ];
+
+  meta = with lib; {
+    description = "A light-weight integrated development environment for the ChucK digital audio programming language";
+    homepage = "http://audicle.cs.princeton.edu/mini/";
+    downloadPage = "http://audicle.cs.princeton.edu/mini/linux/";
```
Comment (SuperSandro2000) [inline pkgs/applications/audio/miniaudicle/default.nix:53]:
```suggestion
    homepage = "https://audicle.cs.princeton.edu/mini/";
    downloadPage = "https://audicle.cs.princeton.edu/mini/linux/";
```

Comment (fgaz) [reply pkgs/applications/audio/miniaudicle/default.nix]: Both done

**Comment by ehmry** [comment]
Thanks! This is a nice one to have.

**Pull request merged**